### PR TITLE
zh-trans: update docs/reference/kubectl/kubectl.md

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by chenrui333** [comment]
Forced update after rebase to `release-1.12`

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 1]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+<!-- * [kubectl options](kubectl_options.md)	 - Print the list of flags inherited by all commands -->
+* [kubectl options](kubectl_options.md)	 - 打印所有命令行共有的参数
+<!-- * [kubectl patch](kubectl_patch.md)	 - Update field(s) of a resource using strategic merge patch -->
+* [kubectl patch](kubectl_patch.md)	 - 使用重要兼容的补丁来更新一个资源中的字段信息
+<!-- * [kubectl plugin](kubectl_plugin.md)	 - Runs a command-line plugin -->
+* [kubectl plugin](kubectl_plugin.md)	 - 运行命令行插件
+<!-- * [kubectl port-forward](kubectl_port-forward.md)	 - Forward one or more local ports to a pod -->
+* [kubectl port-forward](kubectl_port-forward.md)	 - 将一个或者多个本地端口转发到 pod
+<!-- * [kubectl proxy](kubectl_proxy.md)	 - Run a proxy to the Kubernetes API server -->
+* [kubectl proxy](kubectl_proxy.md)	 - 运行一个 kubernetes API server 代理
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
* [kubectl proxy](kubectl_proxy.md)	 - 运行一个 kubernetes API 服务器代理
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 2]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+<!-- * [kubectl replace](kubectl_replace.md)	 - Replace a resource by filename or stdin -->
+* [kubectl replace](kubectl_replace.md)	 - 通过文件名或控制台标准输入替换资源
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
* [kubectl replace](kubectl_replace.md)	 - 基于文件名或标准输入替换资源
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 3]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+<!-- * [kubectl rollout](kubectl_rollout.md)	 - Manage the rollout of a resource -->
+* [kubectl rollout](kubectl_rollout.md)	 - 完成一个 deployment 的部署
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
* [kubectl rollout](kubectl_rollout.md)	 - 管理资源的上线
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 4]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+<!-- * [kubectl run](kubectl_run.md)	 - Run a particular image on the cluster -->
+* [kubectl run](kubectl_run.md)	 - 在集群中使用指定镜像启动容器
+<!-- * [kubectl scale](kubectl_scale.md)	 - Set a new size for a Deployment, ReplicaSet, Replication Controller, or Job -->
+* [kubectl scale](kubectl_scale.md)	 - 为一个 Deployment, ReplicaSet, ReplicationController, 或者 Job 设置一个新的规格
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md]:
```suggestion
* [kubectl scale](kubectl_scale.md)	 - 为一个 Deployment、ReplicaSet、ReplicationController 或 Job 设置一个新的规模尺寸值
```

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 5]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+      <td style="line-height: 130%; word-wrap: break-word;">TLS 的客户端证书的文件路径</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">--client-key string</td>
+    </tr>
+    <tr>
+      <td></td>
+      <!-- <td style="line-height: 130%; word-wrap: break-word;">Path to a client key file for TLS</td> -->
+      <td style="line-height: 130%; word-wrap: break-word;">TLS 客户端秘钥文件的路径</td>
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/reference/kubectl/kubectl.md]: 谢谢！

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 6]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+      <td style="line-height: 130%; word-wrap: break-word;">log 到标准输出以及文件上</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">--as string</td>
+    </tr>
+    <tr>
+      <td></td>
+      <!-- <td style="line-height: 130%; word-wrap: break-word;">Username to impersonate for the operation</td> -->
+      <td style="line-height: 130%; word-wrap: break-word;">以某人名义进行操作</td>
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/reference/kubectl/kubectl.md]: 其实 这个 感觉差不太多？

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 7]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+      <td colspan="2">--azure-container-registry-config string</td>
+    </tr>
+    <tr>
+      <td></td>
+      <!-- <td style="line-height: 130%; word-wrap: break-word;">Path to the file containing Azure container registry configuration information.</td> -->
+      <td style="line-height: 130%; word-wrap: break-word;">包含容器仓库配置信息的文件路径。</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">--cache-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/Users/zarnold/.kube/http-cache"</td>
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/reference/kubectl/kubectl.md]: 您这个 真是细致。我要不两边都改了吧？

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 8]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+      <td style="line-height: 130%; word-wrap: break-word;">所要使用的 kubeconfig 上下文名</td>
+    </tr>
 
+    <tr>
+      <td colspan="2">--default-not-ready-toleration-seconds int&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 300</td>
+    </tr>
+    <tr>
+      <td></td>
+      <!-- <td style="line-height: 130%; word-wrap: break-word;">Indicates the tolerationSeconds of the toleration for notReady:NoExecute that is added by default to every pod that does not already have such a toleration.</td> -->
+      <td style="line-height: 130%; word-wrap: break-word;">指明 `notReady:NoExecute` 的容忍时间（`tolerationSeconds`），将默认添加到各个没有这个容忍设置的 pod 上。</td>
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/reference/kubectl/kubectl.md]: 👍

File: content/zh/docs/reference/kubectl/kubectl.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/reference/kubectl/kubectl.md]: it deserves an upstream code change: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go#L297

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 9]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+      <td style="line-height: 130%; word-wrap: break-word;">Kubernetes API 服务器的地址和端口</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">--stderrthreshold severity&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 2</td>
+    </tr>
+    <tr>
+      <td></td>
+      <!-- <td style="line-height: 130%; word-wrap: break-word;">logs at or above this threshold go to stderr</td> -->
+      <td style="line-height: 130%; word-wrap: break-word;">日志等级，等于或高于这个等级的日志都将会打印到标准错误输出上</td>
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/reference/kubectl/kubectl.md]: 👍

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 10]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+      <td style="line-height: 130%; word-wrap: break-word;">打印版本信息，然后退出</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">--vmodule moduleSpec</td>
+    </tr>
+    <tr>
+      <td></td>
+      <!-- <td style="line-height: 130%; word-wrap: break-word;">comma-separated list of pattern=N settings for file-filtered logging</td> -->
+      <td style="line-height: 130%; word-wrap: break-word;">指定输出日志的模块，格式如下：pattern=N，使用逗号分隔</td>
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/reference/kubectl/kubectl.md]: 👍

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 11]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+      <td style="line-height: 130%; word-wrap: break-word;">日志等级，等于或高于这个等级的日志都将会打印到标准错误输出上</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">--token string</td>
+    </tr>
+    <tr>
+      <td></td>
+      <!-- <td style="line-height: 130%; word-wrap: break-word;">Bearer token for authentication to the API server</td> -->
+      <td style="line-height: 130%; word-wrap: break-word;">Bearer 令牌，供 API 服务器验证之用</td>
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/reference/kubectl/kubectl.md]: 👍

**Comment by chenrui333** [comment]
@tengqm Thanks a lot for the exhaustive and detailed suggestions, all looks good to me and better, so I adopt all!

Please give another look!

File: content/zh/docs/reference/kubectl/kubectl.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/reference/kubectl/kubectl.md]: You mean this `md` file is a generated file from the code? I was thinking about just to update the documentation side.

[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/reference/kubectl/kubectl.md]: Yes. Reference doc is generated using kubernetes-incubator/reference-docs project

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 12]
```diff
@@ -7,172 +7,399 @@ notitle: true
...
+      <td style="line-height: 130%; word-wrap: break-word;">要求服务器的版本和客户端版本一致</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">-n, --namespace string</td>
+    </tr>
+    <tr>
+      <td></td>
+      <!-- <td style="line-height: 130%; word-wrap: break-word;">If present, the namespace scope for this CLI request</td> -->
+      <td style="line-height: 130%; word-wrap: break-word;">如果指明命名空间，将供 CLI 请求之用</td>
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/reference/kubectl/kubectl.md]: namespace, 是做 命名空间  还是 名字空间？

**Pull request merged**